### PR TITLE
Adding an ini file for l2tdevtools as well as deleting an old dependency file

### PR DESCRIPTION
**Pull request by kiddinn**
Small PR for l2tdevtools integration.

Next up to fix some of the tests, will be required to get the devtools to properly support the project.

### CODE REVIEW

File: PlasoScaffolder.ini [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name: PlasoScaffolder
+name_description: a scaffolder project for plaso, easing parser and plugin creation. 
+maintainer: Log2Timeline maintainers <log2timeline-maintainers@googlegroups.com>
+homepage_url: https://github.com/log2timeline/PlasoScaffolder
+description_short: Plugin and parser scaffolder for plaso.
+description_long: The plaso (log2timeline) scaffolder is a tool written to make
+    writing a new plugin or parser in plaso easier by asking the developer few
+    questions and automatically create all the boilerplate code necessary to
+    start the process.
+python3_only: true
```
Comment (Onager) [inline PlasoScaffolder.ini]: As joachim mentioned, there's no "python3_only", so please remove this

Comment (kiddinn) [reply PlasoScaffolder.ini]: well, there is now... or at least once the other PR gets approved

Comment (kiddinn) [reply PlasoScaffolder.ini]: we can hold off on this until that PR get's pushed in l2tdevtools

**Comment by kiddinn** [comment]
changes made to l2tdevtools to support the python3 definition

File: PlasoScaffolder.ini (hunk 1 above)
Comment (kiddinn) [reply PlasoScaffolder.ini]: PR pushed

**Pull request merged**